style.css

```diff
@@ -534,6 +647,23 @@ html {
   color: #1a2236;
 }
 
+#error {
```

Comment:
[OPTIONAL]
- [ ] You may consider sticking to the design as much as possible. Kindly make sure your form inputs are placed as is the case in the template provided in Figma
Your template :point_down: 
![image](https://user-images.githubusercontent.com/97100091/202839521-67557008-1345-4bb3-9436-99bb7c50caf0.png)

Figma design :point_down: 
![image](https://user-images.githubusercontent.com/97100091/202839501-5d50e52d-d966-4613-9208-5e9543910bc0.png)